EIPS/eip-3664.md

```diff
@@ -0,0 +1,418 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends [ERC-1155](./eip-1155.md)
+author: TylerZ (@tylerztl), TY (@nimocat)
+discussions-to: https://ethereum-magicians.org/t/eip-3664-next-generation-game-nft-standard/6738
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Abstract
+In this proposal, we created the next gen component-NFT standard, which makes up for the problem that ERC-721 and ERC-1155 can only support unchangeable attributes of NFTs. component-NFT can make the attributes in a NFT have five characteristics: transferable, upgradeable, changeable, combinable and evolvable. By using ERC-3664 standard NFTs in any fields, users can create any valueable dapps based on component-NFT standard.
+
+## Motivation
+In the early days of NF, ERC-721 was the main NFT protocol. It focused on differentiating NFTs from FTs. But since the emergence of the Layer 2 and Metaverse, outdated NFT protocols would no longer suffice for NFT standards in the new era. We hope to objectize NFTs, and allow for their upgrades, assembly and disaggregation on the blockchain. We will also let them change with the passage of time so that these fundamental functionalities will endow NFTs with a world of possibilities, creating unparalleled value.
```

Comment:
```suggestion
In the early days of NFTs, ERC-721 was the main NFT protocol. It focused on differentiating NFTs from FTs. But since the emergence of the Layer 2 and Metaverse, outdated NFT protocols would no longer suffice for NFT standards in the new era. We hope to objectize NFTs, and allow for their upgrades, assembly and disaggregation on the blockchain. We will also let them change with the passage of time so that these fundamental functionalities will endow NFTs with a world of possibilities, creating unparalleled value.
```
Spelling error?

Reply:
Also note, ERC-721 isn't a protocol. It's a standard.